invenio_communities/permissions.py

```diff
@@ -74,7 +74,8 @@ class CommunityPermissionPolicy(BasePermissionPolicy):
         ),
     ]
 
-    can_direct_publish = [
+    # who can include a record directly, without a review
+    can_include_directly = [
```

Comment:
There are a couple of occurrences of this that should also be updated, see https://github.com/inveniosoftware/invenio-communities/blob/master/invenio_communities/assets/semantic-ui/js/invenio_communities/community/communitiesItems/CommunityCompactItemComputer.js#L60-L63

Reply:
Thank you, no idea how I missed those 👀